src/pages/BrowseSchemes.svelte

```diff
@@ -39,24 +39,43 @@
   // TODO Extract into a component
   let counts = { area: 0, route: 0, crossing: 0, other: 0 };
 
+  // Dropdown filters
```

Comment:
Reminder that TS checks are disabled for this file, and everything in here is a bit of a hack (not split into distinct components for one). I have ideas about improving the code, but would like a little confidence we'll end up using/developing this before investing the effort

Reply:
In that case maybe this can remain an experimental branch rather than merging it in?

Reply:
Hmm, do you mean moving the current browse page out of `main`? I would prefer keeping it in main and continuing to iterate on it (also in the govuk CSS PR). I think by early next week, we'll have a better idea how important this page is. If it's unlikely we'll use it, we can remove it entirely, and if more features will get added in the short term, we can improve the code. But before then, I'm trying to clean the page up before some meetings, and make it look like govuk style

Reply:
Yeah I suppose since there's already plenty of this on main we can just price in a refactor down the line. I did mean just not merging these changes in before Monday but I guess given we want to show the gov uk stylings as well it's worth

Reply:
Ah, this PR is intentionally here for a Monday demo, to make possible ideas around an inspector view more tangible :)  So if you have other ideas for this page that would be helpful to prototype quickly or even just sketch up / discuss, I'd say they're worth it